index.js

```diff
@@ -458,6 +458,10 @@ module.exports = {
       return callback(null);
     };
 
+    // This method can be overridden to exclude
+    // specific docs from sitemap.
+    self.include = doc => true
```

Comment:
arrow function is OK but we need a semicolon to follow our conventions.

Reply:
arf, why did my linter say nothing...

Reply:
we don't have an eslint file in this project, that's why